queries.sql

```diff
@@ -76,5 +76,21 @@ SELECT OWNERS.full_name, ANIMALS.name AS name_of_animals, SPECIES.name AS name_o
 
 SELECT name, escape_attempts, full_name FROM ANIMALS JOIN OWNERS ON OWNERS.id = ANIMALS.owner_id WHERE OWNERS.full_name = 'Dean Winchester' AND ANIMALS.escape_attempts = 0;
 
-SELECT full_name, COUNT(animals.owner_id) AS Owns FROM OWNERS JOIN ANIMALS ON ANIMALS.owner_id = OWNERS.id GROUP BY full_name ORDER BY Owns DESC;
+SELECT full_name, COUNT(ANIMALS.owner_id) AS Owns FROM OWNERS JOIN ANIMALS ON ANIMALS.owner_id = OWNERS.id GROUP BY full_name ORDER BY Owns DESC;
 
+
+SELECT ANIMALS.name FROM VISITS JOIN ANIMALS ON animals_id=ANIMALS.id WHERE vets_id=1 ORDER BY date_of_visit DESC LIMIT 1;
+
+SELECT * FROM VISITS WHERE vets_id=3;
```

Comment:
- [ ] Kindly alter this query since you need to find the DISTINCT animals that the vet visited, this just returns all of the visits, even if it is with the same animal twice.

Reply:
@zhadier , Noted thank you, let me fix it right away.

Reply:
I have fixed the query.